tracee-ebpf/main.go

```diff
@@ -92,16 +92,10 @@ func main() {
 
 			// environment capabilities
 
-			selfCap, err := getSelfCapabilities()
+			err = ensureCapabilities()
 			if err != nil {
 				return err
 			}
-			if err = checkRequiredCapabilities(selfCap); err != nil {
-				return err
-			}
-			if err = dropUnnecessaryCapabilities(selfCap); err != nil {
```

Comment:
See ? I just reviewed this part and now its gone to somewhere else. (Thus the reason for the request of stashing commits that are playing the same cards).